src/NetscriptFunctions.ts

```diff
@@ -744,21 +744,20 @@ export const ns: InternalAPI<NSFull> = {
     },
   spawn:
     (ctx) =>
-    (_scriptname, _thread_or_opt = 1, ..._args) => {
+    (_scriptname, _thread_or_opt = 1, _spawndelay, ..._args) => {
```

Comment:
You forgot to revert this part. I'm surprised TypeScript didn't yell at you... I guess because of the varargs `...args` that comes after?

Reply:
I'm surprised as well lol. I've sorted this. :+1: